commands/deploy.go

```diff
@@ -250,3 +252,45 @@ func triggerUpdate(accountID, appID int, payloadID, serviceURL string, c *http.C
 	}
 	return nil
 }
+
+// newFileUploadRequest builds a HTTP request for uploading an app and the account + app it belongs to
+func newFileUploadRequest(c *DeployCmd, f *os.File) (r *http.Request, err error) {
+	contents, err := ioutil.ReadAll(f)
```

Comment:
Has this just re-introduced that the cli needs to fit the entire payload into ram?

Reply:
This is likely a better pattern: https://gist.github.com/ZenGround0/49e4a1aa126736f966a1dfdcb84abdae

Reply:
💯  I'm working on an optimization for this now.

Reply:
I'm going to create a separate PR for the memory optimizations, in the `make-file-upload-memory-efficient` branch.

Right now, we just want `sectionctl` to work. Then we can focus on making it right.